modules/sugar/src/main/ts/ephox/sugar/api/search/Traverse.ts

```diff
@@ -67,6 +67,27 @@ const parents = (element: SugarElement<Node>, isRoot?: (e: SugarElement<Node>) =
   return ret;
 };
 
+const parentsElement = (element: SugarElement<Element>, isRoot?: (e: SugarElement<Element>) => boolean): SugarElement<Element>[] => {
```

Comment:
I don't think we need this since there is a `PredicateFilter.ancestors` function that filters and stops of a root. If you can everything down to a root you just apply that with `Fun.always` to get them all.